### ISSUE TITLE
[ui] Dashboards page shows no entries with an error in JS console

### ISSUE THREAD

**Issue by vvv** [zinclabs/zincobserve#548]
### Which ZincObserve functionalities are the source of the bug?

dashboards

### Is this a regression?

Yes

### Steps to reproduce

1. `rm -r data` — this step was necessary in my case (see Note-1 below)
2. Start ZincObserve
3. Create a dashboard. Preferably using HTTP API call (Postman/Insomnia), though UI will do also.
4. Go to/refresh the Dashboards page.

**Note-1:** `cargo run` failed until I removed data/ directory.

```
[2023-04-12T08:43:42Z INFO  zincobserve::service::db::alerts] [TRACE] Start watching alerts
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: Error("missing field `params`", line: 1, column: 163)', /Users/vvv/zn/zincobserve/src/service/db/functions.rs:127:65
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Error: channel closed
```

### Actual result

- The page shows spinner for quite a while
- No dashboards are shown
- There is a `TypeError: Cannot read properties of undefined (reading 'length')` in the JS console

![image](https://user-images.githubusercontent.com/17462/231408215-7192794f-b827-42eb-9a77-54a8c47d8e91.png)

### Version

784255d716e8cf25ecc43e6be9b90c4d82c6d39b

**Comment by ktx-kirtan** [comment]
Thanks @vvv for the detailed steps. Will update you soon

**Comment by vvv** [comment]
This problem is not reproducible with the latest ‘master’. I reckon it was fixed with #539.
